src/extension/options-page/DashboardInitSteps/Step2S.tsx

```diff
@@ -21,8 +21,11 @@ export default function InitStep2S() {
     const personas = useMyPersonas()
     const [persona, setPersona] = useState<Persona | null>(null)
     useAsync(async () => {
-        if (identifier)
-            Services.Identity.queryPersona(Identifier.fromString(identifier, ECKeyIdentifier).unwrap()).then(setPersona)
+        if (identifier) {
+            setPersona(
```

Comment:
refactor to multiple lines is good for reading

```suggestion
            const unwrapedIdentifier = Identifier.fromString(identifier, ECKeyIdentifier).unwrap();
            const persona = await Services.Identity.queryPersona(unwrapedIdentifier);
            setPersona(persona);
```

Reply:
I second this refactoring to improve the code readability.

Reply:
Looks good. I feel as if I can actually read it now!